frameworks/C++/cinatra/cinatra.dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+
+RUN apt-get update -yqq && \
+	apt-get install -yqq cmake git uuid-dev gcc g++ autoconf
+
+ENV IASIO=/asio/asio
+ENV CINATRA_HOME=/cinatra
+ENV CINATRA=/cinatra/example
+
+RUN git clone https://github.com/chriskohlhoff/asio.git
+
+WORKDIR $IASIO
+
+RUN ./autogen.sh && ./configure
+RUN make -j && make install
+
+WORKDIR /
+
+RUN git clone https://github.com/qicosmos/cinatra.git
```

Comment:
Please `git checkout` a specific release or git hash for all repos.

Reply:
ok, have done.